app/src/main/res/layout/topic_overview_fragment.xml

```diff
@@ -72,51 +72,58 @@
       <TextView
         android:id="@+id/see_more_text_view"
         android:layout_width="0dp"
-        android:layout_height="wrap_content"
-        android:layout_marginStart="24dp"
-        android:layout_marginTop="16dp"
-        android:layout_marginEnd="24dp"
-        android:gravity="start"
+        android:layout_height="48dp"
```

Comment:
Suggest using `minHeight` rather than this, or make this `wrap_content` and make adjustment using padding.

Reply:
This text is constant its not going to change so provided fixed height as `48dp`.

Reply:
Done as suggested.